### PR TITLE
add RBR Virtuoso Tu processing to the rsk portion of stglib.

### PR DESCRIPTION
**Pull request by mmartini-usgs**
add code to the appropriate methods in the rsk directory
add scripts, jupyter-notebook for same to scripts
add unit tests
amend setup.py to add scripts and one missing required package (sqlite3)

### CODE REVIEW

**Comment by mmartini-usgs** [comment]
The failed check above seems to do with some things I tried to make more consistent with actual functionality, you can just revert to the original, it should not make a difference to my code if it's a headache for others:

setup.py 

- since sqlite3 was a prerequisite for processing .rsk files, it should be mentioned in the install_requires comment
- I added scripts to the scripts directory so I added them to the scripts list here

cmd.py
- for now more generic rsk code, I removed d|wave specificity from the description

**Comment by dnowacki-usgs** [comment]
Thanks Marinna, looking at this today.

File: scripts/runRBRTurskcdf2nc.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+import sys
+sys.path.append('C:\\projects\\python\\stglib')
```
Comment (dnowacki-usgs) [inline scripts/runRBRTurskcdf2nc.py]: Please don't hard code paths for importing. The assumption is that the user will have installed these packages and are available to python. Same goes for other run scripts.

File: setup.py [hunk 2]
```diff
@@ -22,7 +22,7 @@
       license='Public domain',
       classifiers=['Development Status :: 3 - Alpha',
                    'Programming Language :: Python :: 3.6'],
-      # install_requires=['numpy', 'netCDF4', 'xarray'],
+      # install_requires=['numpy', 'netCDF4', 'xarray', 'sqlite3'],
```
Comment (dnowacki-usgs) [inline setup.py]: sqlite3 is part of the default python install and does not need to be in the requires line

File: stglib/tests/test_RBR_VTu.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+import os
+# the following three lines are necessary to run in an environment
+# where stglib is not on the path
+import sys
+filepath = os.path.join(os.path.dirname(os.path.abspath(__file__)), 
+                        '..\\..\\..\\stglib')
+sys.path.append(filepath)
+# xml to dict is a required module in order to import stglib.  
+# As it is not a part of Anaconda's IOOS distribution, 
+# nor is it likely to be, you may need this
```
Comment (dnowacki-usgs) [inline stglib/tests/test_RBR_VTu.py]: You won't (shouldn't) need this if stglib is installed in the path, which it should be. Please remove these hard-coded lines especially for the tests and they should work. Also please do not use double-escaped backslashes to specfy path, use forward slashes (/) instead, which should still work fine on a windows python install.

**Comment by mmartini-usgs** [comment]
I am closing this pull request because the main code is now too different for a convenient merge.

**Pull request closed without merging**